### PR TITLE
Homework #11

### PR DESCRIPTION
**Pull request by AndryNosach**
Done:
1. Implemented quiz passing!
2. Authorization filter changed to Spring security.

Deployed version: https://my-quiz-project.herokuapp.com/login

### CODE REVIEW

File: Nosach/web-quiz-project/src/main/java/config/QuizConfiguration.java [hunk 1]
```diff
@@ -18,22 +22,19 @@
...
         BasicDataSource dataSource = new BasicDataSource();
         dataSource.setDriverClassName("com.mysql.jdbc.Driver");
-        dataSource.setUrl("jdbc:mysql://localhost:3306/quiz-project?useUnicode=true" +
-                                                                    "&characterEncoding=UTF8" +
-                                                                    "&useJDBCCompliantTimezoneShift=true" +
-                                                                    "&useLegacyDatetimeCode=false" +
-                                                                    "&serverTimezone=UTC");
-        dataSource.setUsername("root");
-        dataSource.setPassword("root");
+        dataSource.setUrl("jdbc:mysql://lt80glfe2gj8p5n2.chr7pe7iynqr.eu-west-1.rds.amazonaws.com:3306/vt8gqxfgfiqppxq6?reconnect=true");
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/config/QuizConfiguration.java:34]: Commiting your urls with credentials to public repository is not good idea

File: Nosach/web-quiz-project/src/main/java/controllers/QuizController.java [hunk 2]
```diff
@@ -49,7 +80,7 @@ public String showQuizAddingForm(Model model){
     }
 
     @RequestMapping(method = POST, value = "add")
-    public String addQuizBasicInfo(HttpSession session, WebRequest req){
+    public String addQuizBasicInfo(HttpSession session, WebRequest req, Principal principal){
```
Comment (dinamio) [inline Nosach/web-quiz-project/src/main/java/controllers/QuizController.java:83]: Do you need this session now?

**Review by dinamio — APPROVED** [review]
Good work, approved

File: Nosach/web-quiz-project/src/main/java/config/QuizConfiguration.java (hunk 1 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/config/QuizConfiguration.java:34]: Ok, will fix this in next commit

File: Nosach/web-quiz-project/src/main/java/controllers/QuizController.java (hunk 2 above)
Comment (AndryNosach) [reply Nosach/web-quiz-project/src/main/java/controllers/QuizController.java:83]: I'm using session attribute in this method. I can change session to @SessionAttribute.
Will fix this in next commit.

**Pull request merged**